server/schema.ts

```diff
@@ -266,3 +266,10 @@ export interface IAdminTemplate extends ICommonTemplate {
 		};
 	};
 }
+
+export type DataLog = {
```

Comment:
I've never used typescript before and wasn't sure which one to use. I can change this to an interface for consistency.